src/include/miopen/readonlyramdb.hpp

```diff
@@ -87,7 +87,7 @@ class ReadonlyRamDb
         return record->GetValues(id, value);
     }
 
-private:
```

Comment:
The internal map object is useful in iterating over the class, I dont see any reason to keep it private. I am open to suggestions though.

Reply:
I'd rather add a getter. Or it would also be editable form outside.

Reply:
Done